### PR TITLE
reflect changes in add-cluster.sh script

### PR DESCRIPTION
**Pull request by MatousJobanek**
to reflect & test changes from https://github.com/codeready-toolchain/toolchain-cicd/pull/102

### CODE REVIEW

File: make/run-cicd-script.mk [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-OWNER_AND_BRANCH_LOCATION=codeready-toolchain/toolchain-cicd/master
+OWNER_AND_BRANCH_LOCATION=matousjobanek/toolchain-cicd/drop-sandbox-yaml-from-add-cluster
```
Comment (MatousJobanek) [inline make/run-cicd-script.mk:2]: reminder to revert this change

**Review by alexeykazakov — APPROVED** [review]

**Review by rajivnathan — APPROVED** [review]
